code/game/machinery/computer/card.dm

```diff
@@ -161,6 +161,7 @@
 	data["scan_name"] = scan ? scan.name : "-----"
 	data["authenticated"] = is_authenticated()
 	data["has_modify"] = !!modify
+	data["age"] = modify ? modify.age_id : "-----"
```

Comment:
this many dashes seems odd given that it should be 2 digits

Reply:
there goes the antediluvian RP